examples/smac/recurrent/centralised/run_qmix.py

```diff
@@ -12,6 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Example running QMIX on SMAC"""
```

Comment:
Is qmix considered centralized? Also can you please update the `ReadMe.md` in the examples with the updates to the exmaples.

Reply:
Yes, QMIX and VDN belong to the "centralised training and decentralised execution" paradigm. I will update the README

Reply:
Done.